packages/l1-l3-teleport/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "l1-l3-teleport",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "description": "This tutorial shows how to bridge tokens from L1 to an L3",
+  "scripts": {
+    "initiate-deposit": "ts-node scripts/initiate-deposit.ts",
+    "initiate-eth-deposit": "ts-node scripts/initiate-eth-deposit.ts",
+    "monitor-deposit-status": "ts-node scripts/monitor-deposit-status.ts",
+    "monitor-eth-deposit-status": "ts-node scripts/monitor-eth-deposit-status.ts"
+  },
+  "dependencies": {
+    "godzillaba-arbitrum-sdk": "^3.3.3-teleport",
```

Comment:
should be changed to `@arbitrum/sdk` when relevant code is published

Reply:
Is there a release of @arbitrum/sdk (beta included) that has this code?

Reply:
not at the moment